systemtest/approvals/TestAgentConfig.approved.json

```diff
@@ -3,17 +3,24 @@
         {
             "@timestamp": "dynamic",
             "agent_config_applied": 1,
-            "data_stream.dataset": "apm.internal",
-            "data_stream.namespace": "default",
-            "data_stream.type": "metrics",
+            "data_stream": {
+                "dataset": "apm.internal",
+                "namespace": "default",
+                "type": "metrics"
+            },
+            "ecs": {
+                "version": "8.6.0-dev"
```

Comment:
Is the `ecs.version` going to keep changing? If so, is it possible to store the field as "dynamic"? I think we don't want to assert ecs versions in these tests anyway.

Reply:
We are in control of the version now (previously it was controlled by libbeat) - it's defined at https://github.com/elastic/apm-server/blob/b7fad983076fead56c1ab44c82c42c32ad00b534/Makefile#L21. It won't change often, but you're right that we don't need to assert it in these tests, so I'll update these tests to make it "dynamic".